tests/test_preprints.py

```diff
@@ -727,3 +728,27 @@ def test_no_call_async_update_on_400_failure(self, requests, mock_async, mock_ma
         update_preprint_share(self.preprint)
         assert not mock_async.called
         assert mock_mail.called
+
+class TestPreprintConfirmationEmails(OsfTestCase):
+    def setUp(self):
+        super(TestPreprintConfirmationEmails, self).setUp()
+        self.user = AuthUserFactory()
+        self.write_contrib = AuthUserFactory()
```

Comment:
My read on this was that only the person creating the preprint was emailed. Should there be a test to confirm emails are not sent to the other users? Or vice-versa if I'm reading it wrong.